piety-promoted-v2/modernized/18-chapter-17.adoc

```diff
@@ -406,7 +409,13 @@ for it is he that has plucked my feet out of the miry clay,
 and set them upon a rock that was higher than I; wherefore to him shall the honor, glory,
 and praise be given, who is over all worthy forever and evermore.
 
-Signed--Thomas Mawditt, 1748.`'
+[.signed-section-closing]
+Signed
+
+[.signed-section-signature]
+Thomas Mawditt 1748
```

Comment:
Not sure if the date should just go by his name since there isn't any place he listed too?

Reply:
let's just remove the "Signed" part entirely, and then move the date down below on it's own as a `[.signed-section-context-close]`